src/main/riddl/Product/product.riddl

```diff
@@ -17,7 +17,6 @@ context ProductContext is {
     event: EventId
   }
   type ProductInfo is {
-    sku: SKU,
```

Comment:
SKU is the productId. I am assuming that you are pulling it from ProductInfo and calling out SKU as an attribute in each of the state definitions? But I don't see that change. So where does SKU go?

Reply:
Any EntityID can be found using `context.entityId` once you are inside an entity, so it's redundant to have IDs in state. I can include it in Product entity like we do in the other entities, or remove from everywhere